system_tests/ec2_handler.py

```diff
@@ -51,6 +54,20 @@ def get_resources_to_teardown(self):
         return self.env.handler.ec2_infra_state_delta(
             before=self.before_run, after=current_state)
 
+    @classmethod
+    def clean_all(cls, env):
+        resources_to_be_removed = env.handler.ec2_infra_state()
+        cls.logger.info("Current resources in account: " + str(resources_to_be_removed))
+        cls.logger.info("Hardcoded resources to not remove: " + str(KEYS_TO_NOT_REMOVE_ON_CLEANUP))
+        resources_to_be_removed['key_pairs'].pop(KEYS_TO_NOT_REMOVE_ON_CLEANUP)
+        cls.logger.info("resources_to_be_removed: " + str(resources_to_be_removed))
+        failed = env.handler.remove_ec2_resources(resources_to_be_removed)
+        assert len(failed['instances']) == 0
+        assert len(failed['key_pairs']) == 0
+        assert len(failed['elasticips']) == 0
+        assert len(failed['security_groups']) == 1
+        # This is the default security group which cannot be removed by a user.
```

Comment:
the comment should be placed before the security group condition and not after it